docs/usage/config-overview.md

```diff
@@ -261,7 +261,7 @@ We chose this behavior because:
 
 If Renovate is installed, and you can see a job log, but Renovate is not onboarding your repository, look for `dryRun` in the logs to confirm you are in Silent mode and then change to Interactive mode either at the Repository level or Organization level.
 
-Additionally, if an Organization is installed with "Selected repositories" then the app will change `onboardingNoDeps` to `true` so that an Onboarding PR is created even if no dependencies are detected.
+Additionally, if an Organization is installed with "Selected repositories" then the app will change `onboardingNoDeps` to `enabled` so that an Onboarding PR is created even if no dependencies are detected.
```

Comment:
```suggestion
Additionally, if an Organization is installed with "Selected repositories" then the app will change `onboardingNoDeps` to `"enabled"` so that an Onboarding PR is created even if no dependencies are detected.
```